flow/core/kernel/vehicle/traci.py

```diff
@@ -518,7 +517,7 @@ def get_num_arrived(self):
     def get_arrived_ids(self):
         """See parent class."""
         if len(self._arrived_ids) > 0:
-            return self._arrived_ids[-1]
+            return self._arrived_ids
```

Comment:
why this change? this affects a lot of the codebase. Otherwise LGTM

Reply:
self._arrived_ids was a list storing ids for each step. in this function we just want the ids in the last step ([-1]). Since I don't have a list anymore, I deleted [-1].

Reply:
So if it's not a list anymore why does it have a len method?

Reply:
good catch!